libs/barista-components/core/src/option/option.ts

```diff
@@ -142,10 +148,19 @@ export class DtOption<T> implements Highlightable, AfterViewChecked, OnDestroy {
         this._stateChanges.next();
       }
     }
+    this._ngZone.runOutsideAngular(() => {
+      fromEvent(this._element.nativeElement, 'mousemove')
```

Comment:
could we use mouseover instead? We only want to know when somebody enters the option and not all consecutive move events as well.

Reply:
mousemove with takeuntil destroy is really long –> you would listen the whole component lifecycle for mousemoves.